db/migrate/20230125145545_add_users_to_like.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# AddUsersToLike
+class AddUsersToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ]  I think this is supposed to create the `author_id` and a corresponding index, so the reference should be referred to the author. It can be similar to:

```ruby
add_references :likes, :author, null: false, foreign_key: { to_table: 'users' }
```

Please kindly fix that 🙃 🙃